provider/oauth2/forms.py

```diff
@@ -336,3 +336,10 @@ def clean(self):
 
         data['client'] = client
         return data
+
+
+class ClientCredentialsGrantForm(ScopeMixin, OAuthForm):
+    """
+    Validate a client credentials grant request.
+    """
+    scope = ScopeChoiceField(choices=SCOPE_NAMES, required=False)
```

Comment:
If this is an urgent PR and you don't need support for `scope` right now, you can remove the scope field in the form and not worry about this for now.  However, if you choose to do so, I'd recommend removing the scope field from the form since it implies that we support it.

Per RFC:
```
The authorization server MAY fully or partially ignore the scope requested by the client, based on the authorization server policy or the resource owner's instructions.  
```